asf_public_discourse_home_decarbonisation/analysis/mse/eda_mse_category_data.py

```diff
@@ -0,0 +1,311 @@
+"""
+This script performs exploratory data analysis on Money Saving Expert data, by looking at:
+- A few statistics such as: number of posts collected, number of replies to posts and range of dates;
+- Number of posts per year;
```

Comment:
Unsure if it's just myself, but when I run the script with `python eda_mse_category_data.py`, the "Number of posts per year" figure is not produced?

Reply:
yup, it was a lapse! :D thanks for noticing

Reply:
fixed :)